### ISSUE TITLE
Compilation fails on Raspberry Pi (missing file from `gpu_fft`)

### ISSUE THREAD

**Issue by kerel-fs** [glidernet/ogn-rf#27]
Compilation on a Raspberry Pi 2 fails with the following error message:
```
kerel@host:~/ogn-rf $ make
g++ -Wall -O3 -ffast-math -DVERSION=0.2.6 -mcpu=arm1176jzf-s -mtune=arm1176jzf-s -march=armv6zk -mfpu=vfp -DUSE_RPI_GPU_FFT -o gsm_scan src/gsm_scan.cc src/mailbox.c src/gpu_fft.c src/gpu_fft_twiddles.c src/gpu_fft_shaders.c -lpthread -lm -ljpeg -lconfig -lrt -lfftw3 -lfftw3f -lrtlsdr
/tmp/ccTTa2mh.o: In function `gpu_fft_prepare(int, int, int, int, GPU_FFT**)':
gpu_fft.c:(.text+0x98): undefined reference to `gpu_fft_alloc(int, unsigned int, GPU_FFT_PTR*)'
gpu_fft.c:(.text+0xbc): undefined reference to `gpu_fft_ptr_inc(GPU_FFT_PTR*, int)'
gpu_fft.c:(.text+0x104): undefined reference to `gpu_fft_ptr_inc(GPU_FFT_PTR*, int)'
gpu_fft.c:(.text+0x134): undefined reference to `gpu_fft_ptr_inc(GPU_FFT_PTR*, int)'
gpu_fft.c:(.text+0x158): undefined reference to `gpu_fft_ptr_inc(GPU_FFT_PTR*, int)'
gpu_fft.c:(.text+0x1fc): undefined reference to `gpu_fft_ptr_inc(GPU_FFT_PTR*, int)'
/tmp/ccTTa2mh.o: In function `gpu_fft_execute(GPU_FFT*)':
gpu_fft.c:(.text+0x2ec): undefined reference to `gpu_fft_base_exec(GPU_FFT_BASE*, int)'
/tmp/ccTTa2mh.o: In function `gpu_fft_release(GPU_FFT*)':
gpu_fft.c:(.text+0x2f0): undefined reference to `gpu_fft_base_release(GPU_FFT_BASE*)'
collect2: error: ld returned 1 exit status
Makefile:30: recipe for target 'gsm_scan' failed
make: *** [gsm_scan] Error 1
```

### Possible fix
Add the missing file `gpu_fft_base.c` from [gpu_fft](http://www.aholme.co.uk/GPU_FFT/Main.htm) release 3.0.

**Comment by kerel-fs** [comment]
We're not the first who failed to add this source file, see also
https://github.com/microtony/RTLSDR-Airband/issues/10#issuecomment-76599482

**Comment by kerel-fs** [comment]
Fixed by df68cc7991be874e858792ff3f15535233a21f25.
